packages/nouns-webapp/src/utils/etherscan.ts

```diff
@@ -1,16 +1,17 @@
-import { CHAIN_ID, ETHERSCAN_API_KEY } from '@/config';
 import { sepolia } from 'viem/chains';
 
+import { ETHERSCAN_API_KEY } from '@/config';
+import { defaultChain } from '@/wagmi';
+
 const getBaseURL = (network: number) => {
-  switch (network) {
-    case sepolia.id:
-      return 'https://sepolia.etherscan.io/';
-    default:
-      return 'https://etherscan.io/';
+  if (network === sepolia.id) {
+    return 'https://sepolia.etherscan.io/';
+  } else {
+    return 'https://etherscan.io/';
   }
 };
```

Comment:
could use `defaultChain.blockExplorers.default.url`

Reply:
It's done.